ResearchKit/Common/ORKInstructionStepViewController.m

```diff
@@ -60,6 +60,7 @@ - (void)stepDidChange {
         self.stepView.continueSkipContainer.continueButtonItem = self.continueButtonItem;
         self.stepView.headerView.learnMoreButtonItem = self.learnMoreButtonItem;
         self.stepView.continueSkipContainer.continueEnabled = YES;
+        self.stepView.continueSkipContainer.hidden = self.readOnlyMode;
         
         self.stepView.instructionStep = [self instructionStep];
     }
```

Comment:
The current implementation includes instruction steps in embedded and standalone mode by default. I like the idea of `excludeInstructionSteps`and will add that property